src/Umbraco.Web/PropertyEditors/ValueConverters/ContentPickerPropertyConverter.cs

```diff
@@ -75,10 +77,10 @@ public override object ConvertDataToSource(PublishedPropertyType propertyType, o
         {
             var attemptConvertInt = source.TryConvertTo<int>();
```

Comment:
This ignores the property editor alias (vs. `ConvertSourceToObject`) is it normal?